workflows/lib/transaction-monitor-workflow.js

```diff
@@ -32,6 +39,7 @@ export class TransactionMonitorWorkflow extends WorkflowEntrypoint {
         {
           timeout: `5 minutes`,
           retries: {
+            delay: '10 seconds',
             limit: 3,
           },
```

Comment:
`delay` is required when configuring retries. Is `10 seconds` a reasonable value or should we change it to something else?

@pyropy

Reply:
10 seconds sounds good to me! 👍🏻